docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -1003,16 +1003,16 @@ They can be used for prewhere optimization only if we enable `set allow_statisti
 
     [HyperLogLog](https://en.wikipedia.org/wiki/HyperLogLog) sketches which provide an estimation how many distinct values a column contains.
 
-- `count_min`
+- `CountMin`
```

Comment:
Names of statistics types are case-insensitive, i.e. both

```sql
CREATE TABLE tab
(
    a LowCardinality(Int64) STATISTICS(minmax)
) [...]
```

and

```sql
CREATE TABLE tab
(
    a LowCardinality(Int64) STATISTICS(MiNmAx)
) [...]
```
 work.

Likewise, names of skipping indexes are case-insensitive. In the [corresponding docs](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree#available-types-of-indices), the headings are CamelCase but there is also a "syntax" field where the statistics name is listed lowercase.

Question: Could you please add `Syntax: minmax` etc. to l. 994-1008?